build.sbt

```diff
@@ -21,3 +30,10 @@ lazy val prinz = (project in file("prinz"))
   .settings(
     name := "prinz",
   )
+
+lazy val prinz_sample = (project in file("prinz_sample"))
+  .settings(commonSettings)
+  .settings(
+    name := "prinz-sample",
+  )
```

Comment:
maybe make it as it should be - not a sample, just prinz?

Reply:
Let's leave it as is since `sample` would result in ambiguous `jar` name.